source/includes/api/_interactions.md

```diff
@@ -0,0 +1,528 @@
+    "location": {
+      "confidence": 0,
+      "lat": 42.3659207,
+      "lng": -71.0587026
+    },
+    "successful": false,
+    "survey": {
+      "surveyId": "116473d5-0ebb-4554-a13c-a4ed965c9857",
+      "responses": {
+        "16a51c61-57b5-4a38-9a7d-2b5f56bfb835": {
+          "fields": {},
+          "questionId": "16a51c61-57b5-4a38-9a7d-2b5f56bfb835"
+        },
+        "6209002e-5921-4b62-83bb-35b4dbe62580": {
+          "fields": {"available": "true"},
+          "questionId": "6209002e-5921-4b62-83bb-35b4dbe62580"
+        }
+      }
+    },
+    "timestamp": "2017-08-29T15:55:12.000Z",
+    "unavailable": false
+  },
+  "id": "0b8c5147-45ab-47d2-b91f-5773034f1f6e",
+  "meta": {
+    "created": "2017-12-15T19:46:01.719Z",
+    "createdBy": "11b78eab-8b3e-45e7-804c-4a94045367d4",
+    "etag": "",
+    "isDeleted": false,
+    "modified": "2017-12-15T19:46:01.719Z",
+    "modifiedBy": "11b78eab-8b3e-45e7-804c-4a94045367d4",
+    "resource": "interactions"
+  },
+  "securityGroupId": "d4a1d326-a2c7-415c-bfea-dd9fc84f8d00"
+}
+```
+
+Attribute | Required? | Description
+--------- | --------- | -----------
+id | true | Unique identifier for the object.
+customerId | true | Customer this resource belongs to.
+securityGroupId | true | Security group of this resource.
+
+### data
+
+Attribute | Required? | Description
+--------- | --------- | -----------
+campaignId | true | Campaign identifier this interaction is associated with.
+contactId | true | Contact identifier of this interaction.
+distance | false | Distance measured in miles.
+location | false | Location [details](#data-location).
+successful | false | Flag indicating whether this interaction was successful.
+survey | true | Survey [details](#data-survey).
+timestamp | true | Date and time when this interaction occurred.
+unavailable | true | Flag indicating whether the listed contact person was unavailable.
+unavailableReason | false | The reason specifying why the listed contact person was unavailable.
+walklistId | false | Route identifier this contact is reserved to.
+listId | true | (Read-only field) List identifier this interaction belongs to.
+
+### data/location
+
+Attribute | Required? | Description
+--------- | --------- | -----------
+confidence | false | Quantitative accuracy of this location co-ordinate.
+lat | false | Co-ordinate specifying the north–south position of this location on Earth.
+lng | false | Co-ordinate specifying the east–west position of this location on Earth.
+
+### data/survey
+
+Attribute | Required? | Description
+--------- | --------- | -----------
+surveyId | true | Survey identifier that the interaction consists.
+responses | false | Object containing all the responses for this survey.
+
+### meta
+
+[See documentation](#metadata-object).
+
+## Create an interaction
+
+> Example Request
+
+```http
+POST /interactions HTTP/1.1
+Host: api.polisapp.com
+Content-Type: application/json
+Authorization: Bearer {access_token}
+```
+
+```json
+{
```

Comment:
This `POST` request body will not work. It is not wrapped around `data`. Also, the indentation is not good.